Jenkinsfiles/Jenkinsfile.cbc-run-integration-tests

```diff
@@ -0,0 +1,83 @@
+pipeline {
+  agent {
+    kubernetes {
+      defaultContainer "bb2-cbc-build"
+      yamlFile "Jenkinsfiles/cbc-run-integration-tests.yaml"
+    }
+  }
+
+  environment {
+    DJANGO_USER_ID_SALT = credentials("bb2-integration-tests-bfd-salt")
+    DJANGO_USER_ID_ITERATIONS = credentials("bb2-integration-tests-bfd-iterations")
+    BRANCH = "${params.COMMIT_SHA == "" ? params.BRANCH : params.COMMIT_SHA}"
```

Comment:
This conditional is to set the `BRANCH` if a `COMMIT_SHA` parameter is included (for Github integrations).